CMakeLists.txt

```diff
@@ -68,16 +70,29 @@ include(cmake/Summary.cmake)
 # Print configuration summary
 print_configuration_summary()
 
-# Set version variables
-include(cmake/Common.cmake)
-create_version_vars(hiprand 1 8 2)
-create_version_vars(rocrand 1 8 2)
+# Setup VERSION
+set(VERSION_STRING "2.5.0.")
+
+# Check if BUILD_NUMBER is defined in a Jenkins environment
+if($ENV{BUILD_NUMBER} )
+  string(CONCAT BUILD_VERSION ${VERSION_STRING} $ENV{BUILD_NUMBER})
+else()
+  string(CONCAT BUILD_VERSION ${VERSION_STRING} "0")
+endif()
+
+rocm_setup_version(VERSION ${BUILD_VERSION} NO_GIT_TAG_VERSION)
+set(hiprand_VERSION "${rocrand_VERSION}")
+math(EXPR temp "${rocrand_VERSION_MAJOR} * 100000 + ${rocrand_VERSION_MINOR} * 100 + ${rocrand_VERSION_PATCH}")
```

Comment:
I might have missed this before, do we still need this math expression?

Reply:
No, we need it for rocRAND/hipRAND implementation of https://docs.nvidia.com/cuda/curand/group__HOST.html#group__HOST_1ga8887f51c309a3e3f2541399a774f14d